src/utils/params.ts

```diff
@@ -21,12 +21,16 @@ export function updateUrlCode(code: string) {
 
 export function getParamsFromQuery(location = history.location) {
   try {
-    return new URLSearchParams(
-      playroomConfig.paramType === 'hash'
-        ? location.hash.replace(/^#/, '')
-        : location.search
-    );
-  } catch (err) {
+    // Prefer checking `hash`, fall back to `search` in case the user has configured a custom
+    // `frameSrc` function that sets search params instead of the hash
+    const isParamsInHash = location.hash.startsWith('#?');
+
+    if (isParamsInHash) {
+      return new URLSearchParams(location.hash.slice(1));
+    }
+
+    return new URLSearchParams(location.search);
```

Comment:
Thoughts on flattening this a little. The minor logic change and comment I think help and not sure its easier to read in the expanded form.

```suggestion
    return new URLSearchParams(
      location.hash.startsWith('#?')
        ? location.hash.slice(1)
        : location.search
    );
```